platform/ios/MGLMapView.mm

```diff
@@ -3324,6 +3328,40 @@ void afterRender() override
         const float scaleFactor;
 };
 
+
+class MGLLogObserver : public mbgl::Log::Observer
+{
+public:
+    MGLLogObserver(MGLMapView *nativeView_)
+        : nativeView(nativeView_) {}
+    virtual ~MGLLogObserver() {}
+    
+    bool onRecord(mbgl::EventSeverity severity, mbgl::Event event, int64_t code, const std::string &msg)
+    {
+        __block bool didNotifyDelegate = false;
+        if (severity == mbgl::EventSeverity::Error && event == mbgl::Event::Setup)
```

Comment:
How closely does `mbgl::Event::Setup` correlate to map loading errors (as opposed to networking errors)?